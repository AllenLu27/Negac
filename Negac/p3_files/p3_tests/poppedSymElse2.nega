void fn(){
	if (false){
	} else {
		int c;
	}
	c = 1;
}
