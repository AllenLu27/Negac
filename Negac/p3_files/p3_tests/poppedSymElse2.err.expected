FATAL [6,2]: Undeclared identifier
Semantic analysis failed
